src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -2032,6 +2004,46 @@
   "com.minecolonies.core.townhall.patreon": "Feature Limited to Contributors and Donors",
   "com.minecolonies.coremod.gui.chat.drcat": "Miau, mew, mau, meee, mew, hiss, nyan, mew!!! brrrrrrrrrbrrrrrrbrrrrr",
   "com.minecolonies.coremod.gui.chat.hyrios": "I am Asuna, I have come here from Aincrad to defend your freedom!",
+  "com.minecolonies.job.builder": "Builder",
+  "com.minecolonies.job.deliveryman": "Courier",
+  "com.minecolonies.job.miner": "Miner",
+  "com.minecolonies.job.lumberjack": "Forester",
+  "com.minecolonies.job.farmer": "Farmer",
+  "com.minecolonies.job.undertaker": "Undertaker",
+  "com.minecolonies.job.fisherman": "Fisher",
+  "com.minecolonies.job.baker": "Baker",
+  "com.minecolonies.job.cook": "Cook",
+  "com.minecolonies.job.shepherd": "Shepherd",
+  "com.minecolonies.job.cowboy": "Cowhand",
+  "com.minecolonies.job.swineherder": "Swineherder",
+  "com.minecolonies.job.chickenherder": "Chickenherder",
+  "com.minecolonies.job.ranger": "Archer",
+  "com.minecolonies.job.knight": "Knight",
+  "com.minecolonies.job.composter": "Composter",
+  "com.minecolonies.job.student": "Student",
+  "com.minecolonies.job.archertraining": "Archer Squire",
+  "com.minecolonies.job.combattraining": "Knight Squire",
+  "com.minecolonies.job.sawmill": "Carpenter",
+  "com.minecolonies.job.blacksmith": "Blacksmith",
+  "com.minecolonies.job.stonemason": "Stonemason",
+  "com.minecolonies.job.stonesmeltery": "Stonesmelter",
+  "com.minecolonies.job.crusher": "Crusher",
+  "com.minecolonies.job.sifter": "Sifter",
+  "com.minecolonies.job.florist": "Florist",
+  "com.minecolonies.job.enchanter": "Enchanter",
+  "com.minecolonies.job.researcher": "Researcher",
+  "com.minecolonies.job.healer": "Doctor",
+  "com.minecolonies.job.pupil": "Pupil",
+  "com.minecolonies.job.teacher": "Teacher",
+  "com.minecolonies.job.dyer": "Dyer",
+  "com.minecolonies.job.glassblower": "Glassblower",
+  "com.minecolonies.job.mechanic": "Mechanic",
+  "com.minecolonies.job.fletcher": "Fletcher",
+  "com.minecolonies.job.planter": "Planter",
+  "com.minecolonies.job.rabbitherder": "Rabbitherder",
+  "com.minecolonies.job.concretemixer": "Concrete Mixer",
+  "com.minecolonies.job.beekeeper": "Beekeeper",
+  "com.minecolonies.job.cookassistant": "Assistant Cook"
```

Comment:
Is there some reason the `coremod` part was removed?  Granted it seems a little silly but changing this will break any existing translations...

Reply:
Yes, on one hand, I want to slowly get rid of the "coremod" stuff. And on the other hand are all of those now consistent. I wanted to be able to properly differentiate between new old and if sth breaks where we are still expecting the old it will be easier to find.